### DISCUSSION TITLE
Error Occurred : '异常'

### DISCUSSION THREAD

**Discussion by billgets123** [PaddlePaddle/PaddleSpatial#151] [BaiduKDDCup2022] [answered]
Dear Organizer,
I got an error ' Error Occurred : '异常', My team name is 'hahaha'. Submit Time is 2022-06-06 16:51. I'd like to get the detail of the error message.

**Comment by ramber1836** [comment, marked as answer]
The argument parser is not allowed in the submitted code since it will conflict with the AIStudio eval_main() system call.

<img width="1097" alt="image" src="https://user-images.githubusercontent.com/19561583/172136814-03988ecf-39fa-4f85-ad86-0229031d2bd1.png">

**Reply by billgets123** [reply]
Is it ok to import argparse in the files that won't be executed online? There are some files that only execute in the local environment for training the model, do I need to remove these usage of argparse as well?

**Reply by ramber1836** [reply]
Only the scripts involved in the online test should remove the argparse.
